src/OrchardCore.Modules/OrchardCore.ContentLocalization/Services/CulturePickerOptionsConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell.Configuration;
+
+namespace OrchardCore.ContentLocalization.Services
+{
+    public class CulturePickerOptionsConfiguration : IConfigureOptions<CulturePickerOptions>
```

Comment:
this class is unnecessary

Reply:
May be and may be not, I saw the pattern in `Media` module, so I don't wanna introduce different style in each module

Reply:
It's only required in `MediaOptions` because of the issues with how arrays are managed through `IConfiguration` (they become additive, rather than replacing). you don't need it here